confidential-data-hub/docs/SECURE_STORAGE.md

```diff
@@ -62,3 +62,24 @@ flowchart LR
 ```
 
 For more details, please refer to [the guide](use-cases/secure-mount-with-aliyun-oss.md).
+
+### Block Device
+
+The [plugin](../storage/src/volume_type/blockdevice) provides ways to encrypt a block device and mount it to a specific mount point. Currently only support LUKS in [cryptsetup](https://gitlab.com/cryptsetup/cryptsetup/) for block device encryption.
+
+#### LUKS Method
+
+In this mode, the device would be encrypted as LUKS device first, and then mount it to a target path to store the data to protect the confidentiality and integrity of the data.
+
+The architecture diagram is
+
+```mermaid
+flowchart LR
+    A[Block Device] -- mount --> B[Block Device]
+    subgraph TEE Guest
+        B -- encrypt by cryptsetup --> C[LUKS Block Device]
+        C -- mount --> D[Target Path]
```

Comment:
Could we mark which devices are encrypted, and which are plaintext?

Reply:
Done.